api/database/migrations/2015_11_12_164024_create_ratings_table.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+use App\Models\User;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Spira\Rate\Model\Rating;
+
+class CreateRatingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create(Rating::getTableName(), function (Blueprint $table) {
+            $table->uuid('rating_id')->primary();
+            $table->uuid('rateable_id');
+            $table->string('rateable_type');
```

Comment:
it is faster not due to constraint but due to value, as it is stored as int (just fyi) =))
:+1: